pkg/vm/stackitem/json.go

```diff
@@ -68,6 +68,7 @@ func ToJSON(item Item) ([]byte, error) {
 // It doesn't contain any pointer and uses less memory than `[]byte`.
 type sliceNoPointer struct {
 	start, end int
+	itemsCount int
```

Comment:
And now the structure is exactly of the slice size. But has some additional functionality.

Reply:
Storing just a byte slice won't save as, because we need to store `itemsCount` anyway, so I'd suggest to fix the comment and go with this structure.

Reply:
Yeah, I see that, no real other options.